sal_interfaces/WeatherStation/WeatherStation_Telemetry.xml

```diff
@@ -8,21 +8,21 @@
       <EFDB_Name>ambient_temp</EFDB_Name>
       <Description>The ambient temperature.</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>degree</Units>
+      <Units>deg_C</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>humidity</EFDB_Name>
       <Description>The humidity.</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>unitless</Units>
+      <Units>%</Units>
       <Count>1</Count>
     </item>
     <item>
       <EFDB_Name>pressure</EFDB_Name>
       <Description>The pressure outside.</Description>
       <IDL_Type>double</IDL_Type>
-      <Units>unitless</Units>
+      <Units>pa</Units>
```

Comment:
Meteorologists are using hPa as the standard unit for atmospheric pressure. Maybe switch to hPa?

Reply:
Correct. I will update that because the WeatherStation indeed outputs hPa.